apps/storybook-react-native/tailwind-intellisense.config.js

```diff
@@ -0,0 +1,12 @@
+const designSystemPreset = require('@metamask/design-system-tailwind-preset');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  presets: [designSystemPreset],
+  content: [
+    '../../packages/design-system-react/src/**/*.{js,jsx,ts,tsx}',
+    '../../packages/design-tokens/stories/**/*.{js,jsx,ts,tsx,mdx}',
+    './src/**/*.{js,jsx,ts,tsx}',
+    './stories/**/*.{js,jsx,ts,tsx}',
+  ],
+};
```

Comment:
This tailwind config is only for tailwind-intellisense so we get design token generated classnames which we may also use in mobile